bin/release

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env bash
+
+# Dependencies:
+#
+# - curl
+# - jq
+
+set -eo pipefail
+
+# Terminal colors
+declare -r COLOR_DEFAULT='\033[0;39m'
+declare -r COLOR_ERROR='\033[0;31m'
+declare -r COLOR_INFO='\033[0;36m'
+declare -r COLOR_RESET='\033[0m'
+declare -r COLOR_SUCCESS='\033[0;32m'
+declare -r COLOR_WARNING='\033[0;33m'
+
+# Globals
+declare GH_TOKEN=""
+declare -r GH_ORG="LearningLocker"
+declare -r GH_LRS_PROJECT="LearningLocker"
+declare -r GH_XAPI_PROJECT="xapi-service"
+# MacOSX, GNU/Linux compatibility (-t option is required for Darwin)
+declare -r WRK_DIR=$(mktemp -d 2> /dev/null || mktemp -d -t 'll-release')
+
+
+# usage: display usage with the appropriate exit code
+#
+# usage: usage [EXIT_CODE]
+#
+#   EXIT_CODE: program exit code (default: 0)
+function usage(){
+
+    exit_code="${1:-0}"
+
+    echo "Usage: bin/release [OPTIONS] COMMAND
+
+OPTIONS:
+  -h, --help
+         print this message
+  -t, --token
```

Comment:
it can be boring to set it every time ? Maybe we can do like in [fun-config](https://github.com/openfun/fun-config) repo ?

Reply:
It's not.